source/one-event.ts

```diff
@@ -1,15 +1,19 @@
+import {JestChromeNamespace} from "jest-chrome/types/jest-chrome.js";
```

Comment:
Uhm no, this turns `jest-chrome` into a non-dev dependency, we can't do this. I'd rather suggest changes to `jest-chrome` so that they support the regular `chrome.events.Event` namespace